Free-Code-Camp-JavaScript-Style-Guide.md

```diff
@@ -30,7 +31,7 @@ function foo() {
 }
 
 ```
-Bad: 
+Bad:
 ```js
 var foo = function () {

```

Comment:
I think that the point that is being made was that there shouldn’t be a
space between the word “function” and the parens. It shouldn’t be:

var foo = function () {

but should be:

var foo = function() {

​

On Thu, Nov 19, 2015 at 11:48 AM, Rex Schrader <notifications@github.com>
wrote:

> In Free-Code-Camp-JavaScript-Style-Guide.md
> <https://github.com/FreeCodeCamp/wiki/pull/104#discussion_r45366512>:
>
> > @@ -30,7 +31,7 @@ function foo() {
> >  }
> >
> >  ```
> > -Bad:
> > +Bad:
> >  ```js
> >  var foo = function () {
>
> I think this is a wrong bad example. The curly brace should be on the next
> line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeCodeCamp/wiki/pull/104/files#r45366512>.
>